charts/agent-control/templates/ac-deployment-secret.yaml

```diff
@@ -0,0 +1,11 @@
+{{- if (index .Values "agent-control-deployment").enabled -}}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ include "agent-control.release.name" . }}
```

Comment:
should be  `agent-control.secret.name` instead, i see that is referenced that way in the cli?

Reply:
You are right. It works because the templates contain the same printf function, but this should be the secret. Thanks!!!